api/src/main/kotlin/team/backend/presentation/quiz/dto/QuizDto.kt

```diff
@@ -19,6 +19,6 @@ class QuizDto {
 
     data class AnswerResponse(
         val isCorrect: Boolean,
-        val answer: String,
+        val answer: String?,
```

Comment:
@BAEKJungHo 
If I don't modify the answer field to nullable, error is occured.
```JSON decoding error: Instantiation of [simple type, class team.backend.presentation.quiz.dto.QuizDto$AnswerResponse] value failed for JSON property answer due to missing (therefore NULL) value for creator parameter answer which is a non-nullable type```
Do you know the solution?

Reply:
@mangchhe 

Do you mean when you test directly with the API or is it an error that occurs in the operation of the test code?

Reply:
@mangchhe 
If the error occurred while executing the test code, it could be because of `any()`.

Reply:
@BAEKJungHo 
There is an error in the test code.
I solved it by changing the parameter of expectBody() to none or SuccessResponse::class.java. 
I wonder why test code is successful when expectBody's parameter is not SuccessResponse::class.java in your code.